### ISSUE TITLE
Missing required a11y aria labels

### ISSUE THREAD

**Issue by maxmednik** [Polymer/core-menu#23]
1. Use appropriate aria roles for elements, e.g. role="menu" and role="menuitem". Elements currently use confusing roles like "img" or are missing roles.

2. Use meaningful aria labels, e.g. "arrow-drop-down" is too generic.

3. Use aria-activedescendant property for the actively selected menu item.
